LICENSE

```diff
@@ -0,0 +1,202 @@
+
+                                 Apache License
```

Comment:
The existing code is licensed under BSD. You can't change the license of it without an agreement of all contributors.

Reply:
Whoops, sorry! Trying to pass test_copyright.py tests, but I had removed that awhile ago.